packages/corvid-e2e-tests/test/loginLogout.e2e.js

```diff
@@ -0,0 +1,78 @@
+const tempy = require("tempy");
+const corvidCliDriverCreator = require("./drivers/cliDriver");
+const { getCorvidTestUser } = require("./drivers/utils");
+const connectToLocalEditor = require("./drivers/connectToLocalEditor");
+
+const BLANK_EDITOR_URL =
+  "https://editor.wix.com/html/editor/web/renderer/edit/bdce5f39-c471-4198-9778-c6c8118dbaa8?metaSiteId=d387aea5-55d2-47cf-a348-3d1fbe316245&editorSessionId=bf5f376b-7485-4995-a6c6-d7535d9c7853";
+
+describe("login / logout", () => {
+  let cliDriver;
+
+  beforeEach(async done => {
+    const cwd = tempy.directory();
+    cliDriver = corvidCliDriverCreator({ cwd });
+    await (await cliDriver.logout()).waitForCommandToEnd();
+    done();
```

Comment:
is this necessary?

Reply:
the idea is that **every cli command** will return the **same** cliCommandResult object, which has 2 methods and a property. the methods are:  waiting for the command to end and kill the command process programmatically and the editor debug port as property.